assets/js/utils.js

```diff
@@ -62,34 +61,38 @@ export function logUserEvent(name, metadata) {
 }
 
 export function timeDifference (start, end) {
```

Comment:
This is to replace sugar-date, right? I get that replacing globals is a bad pattern, but given that it was causing no problems and we're going to move away from CB eventually anyway... why this code now?

Reply:
The `timediff` isn't exactly related to `sugar-date`. I deleted `timediff` because I was caught in dependency hell.

`sugar-date` and `timediff` are both broken libraries that rely on `moment`.

Before I worked here, ChartBuilder's dependencies were coded into `package.json` -- I assume that was so we'd nail down old versions. Part of this pull request involves deleting these dependencies that don't belong in `package.json`. So I deleted `sugar-date`. And then I deleted `moment`. And then I noticed ... `timediff` uses `moment`.

`moment` added ~1MB to our development build, because it included all locale data. (It was broken.) Upstream has fixed the bug, but we depended on the broken version. A significant portion of the "win" of deleting ChartBuilder comes from _not_ using this old version of `moment`.

And we were stuck in that same dependency-hell pattern with `timediff`: I didn't know which version of `moment` it needed....

And I learned that `moment` was absolutely unnecessary. So that simplified things: delete `timediff`.